### ISSUE TITLE
meanflow训练

### ISSUE THREAD

**Issue by kamdkslansgka** [Guowei-Zou/dm1-release#3]
https://github.com/Guowei-Zou/dm1-release/blob/2fa7214962d27fb7633559676546210ee010690f/model/flow/shortcut_dispersive.py#L316

作者您好，请问pre_meanflow_dispersive_mlp_img.yaml文件训练的时候，这里的损失为何阶段梯度呢？

**Comment by Guowei-Zou** [comment]
非常感谢您的宝贵意见，这对我来说非常重要！

  经过核对，您发现的问题确实存在。torch.no_grad() 会阻断梯度反向传播，导致 dispersive loss 无法正常作用于模型训练。这是我在发布 release 版本时引入的 bug。

  我已经在最新的 commit 中修复了这个问题：https://github.com/Guowei-Zou/dm1-release/commit/575d8d0

  再次感谢您的反馈！

  ---

  Thank you very much for your valuable feedback — it is greatly appreciated!

  After verification, the issue you identified does exist. The torch.no_grad() blocks gradient backpropagation, causing the dispersive loss to have no effect on model training. This was a bug introduced when I published the release version.

  I have fixed this issue in the latest commit: https://github.com/Guowei-Zou/dm1-release/commit/575d8d0

  Thanks again for your feedback!
